app/src/main/java/com/courseproject/tindar/entities/Conversation.java

```diff
@@ -1,25 +1,43 @@
 package com.courseproject.tindar.entities;
 
 import java.util.ArrayList;
+import java.sql.Timestamp;
 
 public abstract class Conversation {
     private String user1;
     private String user2;
-    private ArrayList<Message> messages;
+    private ArrayList<MessageInterface> messages;
```

Comment:
I think I'm missing something here. 

> Do you want to do something like ArrayList ?
This is an ArrayList, so I am confused.

Reply:
lol I stopped typing in the middle. I was saying do you want to do something like ArrayList<Message> since Message is  contains attributes which stores the data, MessageInterface just contains the definition of the methods.

Reply:
Not sure if I'm misunderstanding, but I think this is fine; Conversation should only need the methods implemented in MessageInterface. If I need to change this then I'll also figure that out in the next PR.

Reply:
Okay it wasn't me stopped typing but it was github doing weird thing with <>. Anyway, we can revisit when you are working on backend.